documentation/embeddable-interactive-filtering.md

```diff
@@ -8,61 +8,46 @@ Interactives on the page bypass this "answer" approach. They use the interactive
 
 The numbering of items in the activity boils down to:
 
-    activity.questions.index(item) + 1
+    activity.reportable_items.index(item) + 1
 
-This happens when individual 'answer' partials call the Embeddable::Answer#question_index to find the index of the 'question' on the page. The Xhtml partial  doesn't look for an index. Reportable interactives technically have an index, but they do not display it.
+This happens when individual 'answer' partials call the `Embeddable::Answer#question_index` to find the index of the 'question' on the page. The Xhtml partial  doesn't look for an index. Reportable interactives technically have an index because they are part of reportable_items, but they do not display it.
```

Comment:
I think this is right.  If we have an interactive that is reportable, with two multiple choice questions `between` it, the question numbers for the multiple choice questions would be say 5 and 7. (skipping one for the interactive that is reportable).  It might be good to have @ddamelin verify this is acceptable behavior.